src/main/java/com/rarchives/ripme/ripper/rippers/ChanRipper.java

```diff
@@ -19,7 +19,7 @@
 
 public class ChanRipper extends AbstractHTMLRipper {
     public static List<ChanSite> explicit_domains = Arrays.asList(
-        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org")),
+        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org", "is2.4chan.org")),
```

Comment:
I wonder if they will keep growing the list of servers here and if we can be more proactive with a regex instead? Might be a bigger change but it would mean the ripper would keep working for longer. (#432, #362)